### DISCUSSION TITLE
Throttle body issue …  can I record LOG?

### DISCUSSION THREAD

**Discussion by DavidIgor** [cedricp/ddt4all#999] [Q&A]
will log file show what is going on with my throttle body or ecu ? my car is dokker h4m 2018. 
if so please point me right direction how to capture log file and who or how to read it?!

**Comment by KarelSvo** [comment]
You should diagnose errors with PyRen.
Connect interface, select your language, click scan, select vehicle type.

**Reply by DavidIgor** [reply]
there are 0 errors

**Comment by KarelSvo** [comment]
Describe the effects of the problem while driving.
Copy the contents of the ID menu into your answer. 
Did the problem arise on its own or was it caused by playing with software?

**Reply by DavidIgor** [reply]
it doesn't react to APP ,TB operates on first 30% of its range ... then it stops at 30% no matter that APP is being pressed more ... when APP fully pressed ECU switches to open fuel loop and TB opens (jumps) to wot (to 100% open). 
I brought my car with this issue and naturally went to try ddt4all as all other diag are not showing anything! (plus I like that ddt4all can do lot more than just diag).

**Comment by KarelSvo** [comment]
Google translate doesn't know your abbreviations. Therefore I cannot understand the problem.

You still haven't shown the contents of the ID menu.

**Reply by DavidIgor** [reply]
AP - accelerator pedal TB -throttle body.... 
what is ID menu?
